tests/publiccloud/prepare_tools.pm

```diff
@@ -50,21 +51,21 @@ sub run {
     zypper_call('ar ' . $tools_repo);
     zypper_call('--gpg-auto-import-keys in python3-ipa python3-ipa-tests git-core');
```

Comment:
Let's add some `-q` and on the other `zypper in` calls.

Reply:
done http://fromm.arch.suse.de/tests/2785